### PR TITLE
Integrate pfs in pf command 

### PR DESCRIPTION
**Pull request by YingChen1996**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: src/promptflow/promptflow/_cli/_pf/_service.py [hunk 1]
```diff
@@ -45,52 +52,100 @@ def get_app(environ, start_response):
...
-    """Add action to start pfs."""
-    start_pfs_parser = subparsers.add_parser(
-        "start",
-        description="Start promptflow service.",
-        help="pfs start",
+def add_service_parser(subparsers):
+    """Add service parser to the pf subparsers."""
+    service_parser = subparsers.add_parser(
+        "service",
+        description="Manage promptflow service.",
```
Comment (wangchao1230) [inline src/promptflow/promptflow/_cli/_pf/_service.py:59]: Introduce a bit on what pf service does:  provide chat and trace ui.

File: src/promptflow/promptflow/_sdk/_service/app.py [hunk 2]
```diff
@@ -142,12 +142,14 @@ def monitor_request():
                         "last_request_time"
                     ] > timedelta(hours=PF_SERVICE_HOUR_TIMEOUT):
                         # Todo: check if we have any not complete work? like persist all traces.
-                        app.logger.warning(f"Last http request time: {app.config['last_request_time']} was made "
-                                           f"{PF_SERVICE_HOUR_TIMEOUT}h ago")
+                        app.logger.warning(
```
Comment (wangchao1230) [inline src/promptflow/promptflow/_sdk/_service/app.py:145]: print decent message when we do start_trace()

Comment (YingChen1996) [reply src/promptflow/promptflow/_sdk/_service/app.py:145]: didn't get it. you mean change in the following pr? https://github.com/microsoft/promptflow/pull/2455/files

File: src/promptflow/promptflow/_cli/_pf/_service.py (hunk 1 above)
Comment (YingChen1996) [reply src/promptflow/promptflow/_cli/_pf/_service.py]: updated

File: src/promptflow/promptflow/_sdk/_service/app.py (hunk 2 above)
Comment (wangchao1230) [reply src/promptflow/promptflow/_sdk/_service/app.py:145]: Yes.

**Review by wangchao1230 — APPROVED** [review]

**Review by crazygao — DISMISSED** [review]

**Review by wangchao1230 — APPROVED** [review]

**Review by wangchao1230 — APPROVED** [review]

**Review by wangchao1230 — APPROVED** [review]

**Review by zhengfeiwang — APPROVED** [review]

**Pull request merged**